app/register-organization/page.tsx

```diff
@@ -0,0 +1,283 @@
+"use client";
+import React, { useState } from "react";
+import { NextPage } from "next";
+import { Button } from "@/components/ui/button";
+import AuthenticationPageTemplate from "@/components/authentication-page-template";
+import { Input } from "@/components/ui/input";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormMessage,
+  FormLabel,
+} from "@/components/ui/form";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+
+import {
+  Command,
+  CommandEmpty,
+  CommandGroup,
+  CommandInput,
+  CommandItem,
+  CommandList,
+} from "@/components/ui/command";
+
+import {
+  Popover,
+  PopoverContent,
+  PopoverTrigger,
+} from "@/components/ui/popover";
+
+import { Check, ChevronDown } from "lucide-react";
+import { cn } from "@/lib/utils";
```

Comment:
kindly use the normal string literal approach as it's being used from the beginning